script/component-migration/samples/AdditionalInfo.jsx

```diff
@@ -0,0 +1,118 @@
+
+export const Sample = () => (
+  <AdditionalInfo
+    className="vads-u-margin-bottom--2"
+    triggerText="Why aren’t all my issues listed here?"
+    disableAnalytics
+  >
+    <p className="vads-u-margin-top--0">
+      If you don’t see your issue or decision listed here, it may not be in our
+      system yet. This can happen if it’s a more recent claim decision. We may
+      still be processing it.
+    </p>
+  </AdditionalInfo>
+);
+
+export const SampleOther = () => (
+  <AdditionalInfo
+    triggerText="Learn more about military base addresses"
+    onClick={null}
+    disableAnalytics
+    status="info"
+  >
+    <span>
+      The United States is automatically chosen as your country if you live on a
+      military base outside of the country.
+    </span>
+  </AdditionalInfo>
+);
+
+export const Sample3 = ({ className = 'sample' }) => (
+  <div className={className}>
+    <div className="vads-u-margin-bottom--2">
+      <AdditionalInfo triggerText="How do I update my personal information?">
+        <h2 className="vads-u-font-size--h5 vads-u-margin-top--3">
+          If you’re enrolled in the VA health care program
+        </h2>
+        <p className="vads-u-margin-y--1">
+          Please contact your nearest VA medical center to update your personal
+          information.
+        </p>
+        <a href="/find-locations/?facilityType=health">
+          Find your nearest VA medical center{' '}
+        </a>
+        <h2 className="vads-u-font-size--h5 vads-u-margin-top--3 vads-u-margin-bottom--1">
+          If you receive VA benefits, but aren’t enrolled in VA health care
+        </h2>
+        <p className="vads-u-margin-y--1">
+          Please contact your nearest VA regional office to update your personal
+          information
+        </p>
+        <a href="/find-locations/?facilityType=benefits">
+          Find your nearest VA regional office
+        </a>
+      </AdditionalInfo>
+    </div>
+    <ProfileInfoTable
+      title="Personal information"
+      data={[
+        { title: 'Date of birth', value: 'N/A' },
+        { title: 'Gender', value: 'N/A' },
+      ]}
+      className="vads-u-margin-bottom--3"
+      level={2}
+    />
+  </div>
+);
+
+export const Sample4 = () => (
+  <div className="vads-u-margin-y--3 available-connected-apps">
+    <AdditionalInfo
+      triggerText={`What other third-party apps can I connect to my profile?`}
+    >
+      To find out what other third-party apps are available to connect to your
+      profile,{' '}
+      <a href="/resources/find-apps-you-can-use" onClick={null}>
+        go to the app directory
+      </a>
+    </AdditionalInfo>
+  </div>
+);
+
+export const Sample5 = () => (
+  <AdditionalInfo
+    triggerText="What if I don’t have a bank account?"
+    onClick={null}
+  >
+    <p>
+      The Veterans{' '}
+      <a
+        href="https://veteransbenefitsbanking.org/"
+        target="_blank"
+        rel="noreferrer"
```

Comment:
```suggestion
        rel="noreferrer"
        aria-label="Benefits Banking Program (VBBP) - Opens in a new window"
```

Reply:
This is a sample file that we use to run the migration script to verify that the targeted component name and props are updated. Did you want this added in?

Reply:
yeah, I think I need to add it into the generator as well